mutmut/__main__.py

```diff
@@ -329,27 +336,35 @@ def main(command, argument, paths_to_mutate, backup, runner, tests_dir,
     if using_testmon:
         copy('.testmondata', '.testmondata-initial')
 
-    if not use_coverage:
-        def _exclude(context):
-            del context
-            return False
-    else:
+    if use_coverage or use_patch_file:
```

Comment:
It might be cleaner to separate these conditions into separate `if` statements and deal with the code duplication by refactoring it out into functions.

Otherwise, just allow the duplication and separate the `if` statements anyways.
e.g.
```python
if use_coverage:
    # use coverage setup
elif use_patch_file:
    # use patch file
```

Having them together makes the branching somewhat hard to understand.

Reply:
I see your point but it's not obvious to me how to improve it. I'll add a comment do explain why they are grouped together for now.